keps/sig-node/4800-cpumanager-split-uncorecache/README.md

```diff
@@ -126,226 +130,68 @@ The figure below highlights performance gain when pod placement is aligned to an
-   - If the required CPUs are less than the size of the **total** amount of CPUs to an uncore cache, scan through each uncore cache index in numerical order starting from the first index, and assign CPUs if there are enough **available** CPUs within the uncore cache group.
-4. If the required amount of CPUs cannot fit within an uncore cache group and there are enough schedulable CPUs on the node, assign cores in numerical order.
-5. Container will not be scheduled on the node if there are not enough CPUs to satisfy the container requirements. 
+The following are examples of the implementation of `takeUncoreCache`. For the example below, a 16-core split uncore cache processor is scheduling the following Guaranteed containers using the static policy with preferAlignCPUsByUncoreCache` enabled and disabled.
+- Reserved CPUs: 0-3
+- Allocatable CPUs: 4-15
+- Container 1: 2 CPUs
+- Container 2: 8 CPUs
 
-In the case where the NUMA boundary is larger than a socket (setting NPS0 on a dual-socket system), the allocatable pool of CPUs does not exapnd beyond the respective socket when NUMA hints are provided in the above scheduling policy. The filtered subset of CPUs will still remain within the socket. 
+![C1-2cpus_C2-8cpus](./images/C1-2cpus_C2-8cpus.png)
 
-In the case where the NUMA boundary is smaller than an uncore cache (enabling Sub-NUMA clustering on a monolithic cache system), the allocatable pool of CPUs does not expand beyong the respective NUMA boundary when uncore cache hints are provided in the above scheduling policy. The filtered subset of CPUs will still remain within the NUMA boundary.
+The following shows the two different CPU assignment outcomes for the default static policy allocation and the `preferAlignCPUsByUncoreCache` option. Assuming both containers are running uncore cache sensitive workloads, Container 1 and Container 2 will suffer from the noisy neighbor effect due to both workloads competing to access Uncore Cache 0. Container 2 will suffer from cross-cache latency as CPUs 6&7 need to access Uncore Cache 1 and CPUs 8-15 need to access Uncore Cache 0.
+
+Uncore Cache sensitive applications can be deployed on the edge where lower core count and more efficient systems are utilized. These applications often requires CPUs less than the total CPUs of an Uncore Cache group. Newer split architecture processors have increased uncore cache size and bandwidth which has helped reduced the noisy neighbor problem. However, the workloads can still be sensitive to cross cache latency. The example below helps illustrate why `takePartialUncore` is important to fully maximize nodes that have less uncore caches available. or the example below, a 16-core split uncore cache processor is scheduling the following Guaranteed containers using the static policy with preferAlignCPUsByUncoreCache` enabled and disabled.
+- Reserved CPUs: 0-1
+- Allocatable CPUs: 2-15
+- Container 1: 4 CPUs
+- Container 2: 4 CPUs
+- Container 3: 4 CPUs
```

Comment:
this seems relevant for the previous explanation about packed vs distributed allocation at uncore level. I'd move that above and refer it here.